### ISSUE TITLE
 NexHud VR Steam Overlay not firing on Oculus Rift CV1

### ISSUE THREAD

**Issue by DimensionDevices** [Nexam/NexHud#3]
Hey there,

 NexHud VR Steam Overlay is not firing on Oculus Rift CV1...

Well, ok it **does** open, but it opens in SteamVR, while Elite opens separately - regardless of whether Elite is launched from Oculus Home _or_ launched from SteamVR itself (or from the desktop)...

It **looks** like NexHud should sit on top of the VR "layer" and overlay onto the game - but it seems to be opening in its own instance and doesn't overlay anything. 

I can switch between Elite and NexHud as applications, but they don't want to play together for some reason.

Any clues?! I've been looking for **some time** and haven't managed to find a solution. There's a lack of data on NexHud, and it looks **great**.

I did find the SteamVR settings that enable "overlays to interact with applications directly", but that doesn't seem to have an effect either way.

**Comment by Nexam** [comment]
Oh man... I didn't updated NexHud in a while...
What i think happen is Nexhud is running in windows overlay mode instead of VR, you can force it to VR (it need SteamVR) in the launcher if you have the last version from sources.
Or you can directly launch NexHud with the arg "vr" to force it.
I was planning to come back on this project for Odyssey but I'm starting a new job soon and I dont have a ton of time sadly.

**Comment by DimensionDevices** [comment]
Thanks for the reply my dude. NexHud launches in VR fine, but it launches as a separate application instead of an overlay. I think the problem comes from Elite: Dangerous now using the Oculus runtimes for the CV1 and SV1 headsets. So, it no longer uses SteamVR - and the kicker is, if you **force** Elite to use SteamVR, it still launches the Oculus Runtimes... :/ So it may not be fixable easily :(

**Comment by Nexam** [comment]
Yes, NexHud **need** steamVR to work properly at this time, so not an easy fix sadly
